api/versions.go

```diff
@@ -28,20 +28,21 @@ import (
 )
 
 const (
-	containersApi    = "containers"
-	subcontainersApi = "subcontainers"
-	machineApi       = "machine"
-	machineStatsApi  = "machinestats"
-	dockerApi        = "docker"
-	summaryApi       = "summary"
-	statsApi         = "stats"
-	specApi          = "spec"
-	eventsApi        = "events"
-	storageApi       = "storage"
-	attributesApi    = "attributes"
-	versionApi       = "version"
-	psApi            = "ps"
-	customMetricsApi = "appmetrics"
+	containersApi          = "containers"
+	subcontainersApi       = "subcontainers"
+	machineApi             = "machine"
+	machineStatsApi        = "machinestats"
+	dockerApi              = "docker"
+	summaryApi             = "summary"
+	statsApi               = "stats"
+	specApi                = "spec"
+	eventsApi              = "events"
+	storageApi             = "storage"
+	attributesApi          = "attributes"
+	versionApi             = "version"
+	psApi                  = "ps"
+	customMetricsApi       = "appmetrics"
+	namespacedContainerApi = "namespacedcontainers"
```

Comment:
I know I said to add a new endpoint, but what do you think about modifying the `containersApi` instead? It looks like it's missing a v2 version, so you could add the v2 version and use the [RequestOptions](https://github.com/google/cadvisor/blob/master/info/v2/container.go#L221) to specify the namespace (the `IdType` field).